client/src/components/ui/PageUnavailable.svelte

```diff
@@ -2,10 +2,15 @@
     import { InvalidSession } from '../../api/error';
 
     export let err = null as Error | null;
+
+    const doctrackAngry = new URL('../../assets/icons/doctrack-angry.svg', import.meta.url);
 </script>
 
 <div>
-    <h2>Sorry, this page isn't available.</h2>
+    <br>
+    <img src="{doctrackAngry.pathname}" alt="Doctrack angry logo" />
```

Comment:
I recommend creating a new component named `doctrackAngry.svelte` instead of adding an inline `img` file.

Reply:
Did the same for DoctrackSus.svelte